src/styles/app-theme/app.typography.js

```diff
@@ -1,105 +1,89 @@
-import palette from './app.pallete.js'
-
 const appTypography = {
   fontFamily: ['Poppins', 'Rubik', '-apple-system', 'Arial', 'sans-serif'].join(','),
   h1: {
-    color: palette.primary[900],
     fontFamily: 'Rubik',
     fontWeight: 800,
     fontSize: '72px',
     letterSpacing: '-1.5px',
     lineHeight: '85px'
   },
   h2: {
-    color: palette.primary[900],
     fontWeight: 600,
     fontSize: '61px',
     lineHeight: '92px',
     letterSpacing: '-0.5px'
   },
   h3: {
-    color: palette.primary[900],
     fontWeight: 500,
     fontSize: '49px',
     lineHeight: '74px'
   },
   h4: {
-    color: palette.primary[900],
     fontWeight: 500,
     fontSize: '35px',
     lineHeight: '53px'
   },
   h5: {
-    color: palette.primary[900],
     fontWeight: 400,
-    fontSize: '20px',
-    lineHeight: '30px'
+    fontSize: '24px',
+    lineHeight: '36px'
   },
   h6: {
-    color: palette.primary[900],
     fontWeight: 500,
     fontSize: '20px',
-    lineHeight: '28px',
+    lineHeight: '30px',
```

Comment:
![image](https://user-images.githubusercontent.com/80056472/169157615-e48ff7e9-d080-4eb2-a8af-e736e5cfb960.png)

lineHeight most be 28px

Reply:
Fixed.